frontend/__mocks__/mutationObserver.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
+
+global.MutationObserver = global.MutationObserver || class {
```

Comment:
Nit: no need for `||` fallback since the test env. needs to use the mocked impl. anyway.

```js
window.MutationObserver = class {
  constructor(callback) {}
  disconnect() {}
  observe(element, initObject) {}
};
```

This is similar to other DOM API mocks like `__mocks__/matchMedia.js` which supplement the basic DOM API support provided by `jsdom`.

(This file won't be used in webpack builds, since [all major browsers support MutationObserver](https://caniuse.com/#search=MutationObserver) already.)

Reply:
done